gradle/libs.versions.toml

```diff
@@ -168,6 +169,8 @@ sun-activation = { group = "com.sun.activation", name = "javax.activation", vers
 kafka-clients = { group = "org.apache.kafka", name = "kafka-clients", version.ref = "kafka" }
 kafka = { group = "org.apache.kafka", name = "kafka_2.12", version.ref = "kafka" }
 curator-test = { group = "org.apache.curator", name = "curator-test", version.ref = "curator"}
+cglib = { group = "cglib", name = "cglib", version.ref = "cglib"}
```

Comment:
Could you add this to our LICENSE-binay and NOTICE-binary?

Reply:
It has been added.

Reply:
OK.